smbackend/settings.py

```diff
@@ -14,6 +14,7 @@
     LANGUAGES=(list, ["fi", "sv", "en"]),
     DATABASE_URL=(str, "postgis:///servicemap"),
     ELASTICSEARCH_URL=(str, None),
+    ELASTICSEARCH_INDEX_PREFIX=(str, 'servicemap'),
```

Comment:
Current standard black-formatting rules prefer double-quotes instead of single.

Reply:
Oh yes. I should have noticed that. IT is now changed in commit 012e53ab53a4902e009bdf43a1e015f9a140bf67